.github/workflows/upgrade_test_constraints.yml

```diff
@@ -156,29 +150,28 @@ jobs:
         run: |
           set -x
           pip install -U uv
-          flags=(--quiet --extra pyqt5 --extra pyqt6 --extra pyside2 --extra pyside6_experimental --extra testing --extra testing_extra --extra optional)
+          cd napari_repo
```

Comment:
I'm assuming this is this directory is your working directory that you are installing packages to.

Reply:
This is the directory where I clone napari repo

Reply:
This is the directory where I clone napari repo